examples/evm/src/main.rs

```diff
@@ -86,7 +86,10 @@ async fn main() {
         .add_input(&to_vec(&zkdb).unwrap())
         .build();
     let mut exec = Executor::from_elf(env, EVM_ELF).unwrap();
-    let session = exec.run().unwrap();
+    let segment_dir = temp_dir();
```

Comment:
I think we should use the crate `tempfile` so that the tempdir gets removed when it goes out of scope.